app/blog/index.html

```diff
@@ -16,24 +16,40 @@
 <body>
   {% include 'templates/topnav.html' %}
 
-  <main class="wrapper">
-    <article>
-      <h1>Blog</h1>
-      <p>The latest goings-on with the Polymer project and in the community.
+  <header class="hero">
+    <div class="wrapper">
+      <div id="logo-lockup" class="hero-title">Blog</div>
+      <p class="hero-caption">The latest goings-on with the Polymer project and in the community.
         We'll update this page to announce major releases and showcase cool stuff
         being built with Polymer. Have an idea for an article?
         <a href="https://github.com/Polymer/polymer-project.org/issues/new?labels=article">Suggest it!</a>
       </p>
+    </div>
+  </header>
 
-      {% for item in articles %}
-      <h2><a href="{{{item.path}}}.html">{{{item.title}}}</a></h2>
-      <div class="byline author">
-        <time pubdate="" datetime="{{{item.published}}}">{{{item.published}}}</time>
+  <div class="wrapper">
+
+    {% for item in articles %}
+    <blog-tile style="display: block; border-top: 1px solid #FF4470; margin-bottom: 16px;">
```

Comment:
`<blog-tile>` is not a custom element - use class name for styling instead, and move styles to templates/base-blog.html.

Reply:
Correction: move styles to main.css

Reply:
sure.